### ISSUE TITLE
Example request body is missing after import operation

### ISSUE THREAD

**Issue by virusdefender** [a85/POSTMan-Chrome-Extension#3283]
## App Details:

```
Postman for Chrome
Version 5.0.2
OS X / x86-64
Chrome 59.0.3071.115
```

## Issue Report:
 > 1. Did you encounter this recently, or has this bug always been there:

recently

> 2. Console logs

No error message in console log

## How to reproduce

 Create a collection and add an example,  the example request body is `raw` with `JSON(application/json)`

![image](https://user-images.githubusercontent.com/4939404/28299242-a31e36c2-6baa-11e7-9baf-2f5e5f6261ae.png)

 Export the collection with v2 format and then import it

 The example request body disappeared. 

![image](https://user-images.githubusercontent.com/4939404/28299283-e04f1b74-6baa-11e7-949c-bae0e727d889.png)

BTW: 

 - If request body is other mode like `form-data`, there is no problem.

**Comment by sdnts** [comment]
Hi @virusdefender, thanks for reporting this. I was able to replicate, and we will get on a fix for this. I'll let you know here when a fix is available.

**Comment by kristerbone** [comment]
Work around until v5.2 is released is to export in v1 format

**Comment by sdnts** [comment]
Guys, can you update to 5.2 now and check if the issue persists?

**Comment by prashantagarwal** [comment]
@kristerbone if you are still seeing this issue feel free to reopen this issue.

**Comment by efulmo** [comment]
I was able to replicate the issue with desktop Postman v5.2.0. I exported my requests to a file, logged out (all requests were deleted from the app), then imported the whole collection back. However, all the request data disappeared! I had a lot of JSON samples there. I am still able to find JSON request bodies in the backup file though. It looks like the data is lost somehow during the import process.

Please reopen the issue!

**Comment by prashantagarwal** [comment]
@efulmo We have got this fixed in 5.4.0 version. Can you update to the latest version of Postman and verify that this is resolved.
